apps/fluent-tester/package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "@fluentui-react-native/tester",
   "version": "0.11.1",
+  "description": "A test app to test FluentUI React Native Components during development",
+  "license": "MIT",
+  "author": "Microsoft <fluentuinativeowners@microsoft.com>",
+  "homepage": "https://github.com/microsoft/fluentui-react-native",
```

Comment:
Not for this review but maybe over a doc day or hackathon we can update all the other components to have this metadata as well